test/ssl-tests/20-cert-select.cnf

```diff
@@ -1856,13 +1856,13 @@ CipherString = DEFAULT
 MaxProtocol = TLSv1.3
 MinProtocol = TLSv1.3
 PrivateKey = ${ENV::TEST_CERTS_DIR}/server-ml-dsa-44-key.pem
-SignatureAlgorithms = mldsa44
+SignatureAlgorithms = mlDsA44
```

Comment:
Shouldn't this be ML-DSA-44 to encourage correct usage?

Reply:
No, the correct usage is the TLS signature scheme name from IANA, which is [`mldsa44`](https://www.ietf.org/id/draft-tls-westerbaan-mldsa-00.html#name-iana-considerations). Mind you, IANA have not yet assigned the code points, and we may need to drop support in 3.5 if still not assigned by beta.